src/components/Nav/components/NavLinks/index.js

```diff
@@ -360,6 +376,32 @@ export default class GordonNavLinks extends Component {
         );
       }
 
+      //Re-enable when adding the link to timesheets page
```

Comment:
Should timesheets be available to guests?

Reply:
It's already unavailable to guests, we might just need to disable the link on the guest view

Reply:
I thought of something else, it might make more sense for us to put the link in the Links dialog instead of in the navbar.
![image](https://user-images.githubusercontent.com/5964594/79077507-41f77680-7cd0-11ea-9f7c-3d08d063ec2c.png)